src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1029,6 +1073,35 @@ private Task<ICosmosDBAccount> CreateCosmosDbAsync()
                     .WithWriteReplication(Region.Create(configuration.RegionName))
                     .CreateAsync(cts.Token));
 
+        private async Task<IMySQLManagementClient> CreateAndProvisionMySQLServerAsync(string subnetId)
```

Comment:
This method should return Server, not IMySQLManagementClient. 
mySQLManagementClient should be created outside and passed in as a parameter.
ExecuteQueriesOnAzureMySqlDb method should take Server as a parameter and not create it inside.
Rename: CreateMySqlServerAndDatabaseAsync

Reply:
fixed in latest commit